src/test/java/org/folio/entlinks/controller/InstanceAuthorityLinksIT.java

```diff
@@ -217,6 +227,24 @@ void updateInstanceLinks_positive_deleteAndSaveLinks_whenHaveDifference() {
       .andExpect(totalRecordsMatch(4));
   }
 
+  @Test
+  @SneakyThrows
+  void updateInstanceLinks_positive_ignoreReadOnlyFields() {
+    var instanceId = randomUUID();
+    var incomingLinks = linksDtoCollection(linksDto(instanceId,
+      Link.of(InstanceAuthorityLinkStatus.ERROR, "test")));
+    doPut(linksInstanceEndpoint(), incomingLinks, instanceId);
+
+    var expectedLinks = linksDto(instanceId,
+      Link.of(InstanceAuthorityLinkStatus.ACTUAL, null));
+
+    var stats = bibStatsCollection(expectedLinks);
+
+    perform(statsGetRequest())
+      .andExpect(statsMatch(hasSize(1)))
+      .andExpect(statsMatch(stats));
```

Comment:
Why do you use the stats endpoint? Could you use the get links endpoint instead and check that all links have ACTUAL status?